app/[locale]/job/FormPayment.tsx

```diff
@@ -52,12 +76,16 @@ export default function FormPayment(params: FormPaymentProps) {
           name="date"
           control={control}
           rules={{ required: {value:true, message: t('add_income_required_field')} }}
-          render={({ field }) => (
+          render={({ 
```

Comment:
this is a buncha churn on getting the datepicker and form validation working. i left them in as a demonstration at how it _could_ work. they don't affect functionality (or, what didn't work before still doesn't work now lolsob)